### PR TITLE
Remove Lookup API and related table storage

### LINKED ISSUE #1218: Remove Lookup API
The purpose of the Lookup ID was to receive the original PII of the request. Since we'll no longer be storing raw PII, returning original PII won't be possible.  

- Lookup API is no longer available for API calls.

### PR DESCRIPTION
**Pull request by echappen**
Closes #1218 

## Notes
- Removes Lookup Table Storage resource generation from IAC scripts. These resources will have to be purged manually from Azure environments. 

- [X] Run IAC against `tts/dev` before merging.

### Removes
- Removes the `GET /lookup_ids/{id}` endpoint from Duplicate Participation API
- Removes `lookup_id` property from response results in `/query` endpoint of Duplicate Participation API

### CODE REVIEW

**Comment by kingcomma** [comment]
A couple general thoughts before I forget:

- We should open up a follow-on ticket to update the Query Tool to work with these new changes
- I'm pretty sure we'll need a one-off chunk of CLI commands to remove dangling resources (storage account for sure, and maybe the APIM API operation?) after the updated IaC is run

**Review by kingcomma — COMMENTED** [review]
A nice weight off our shoulders! Looks like there are just a couple docs which still have references to the Lookup API, otherwise this looks 👍:
- `/docs/quick-start-guide-states.md`
- `/docs/iac.md`

On the planning front, I think this  makes obsolete a whole host of open issues. @cjhskippy would you mind confirming that all these can be closed?
- #1113
- #853
- #867 
- #701 
- #687 
- #628 
- #627
- #577

File: docs/openapi/generated/duplicate-participation-api/openapi.md [hunk 1]
```diff
@@ -60,7 +60,12 @@ Queries all state databases for any PII records that are an exact match to the l
 
 |Name|In|Type|Required|Description|
 |---|---|---|---|---|
-|data|body|[[#/paths/~1query/post/requestBody/content/application~1json/schema/properties/data/items](#schema#/paths/~1query/post/requestbody/content/application~1json/schema/properties/data/items)]|true|none|
+|data|body|[object]|true|none|
```
[the remark this answers is not in the document]
Comment (echappen) [reply docs/openapi/generated/duplicate-participation-api/openapi.md:63]: *Sigh* yeah, still a mystery to me...

**Review by kingcomma — APPROVED** [review]
👍

**Pull request merged**